gen3-integration-tests/services/indexd.py

```diff
@@ -92,3 +93,32 @@ def delete_files(self, guids: list, user="indexing_account"):
                 index.delete_record(guid=guid)
             except Exception as e:
                 logger.exception(msg=f"Failed to delete record with guid {guid} : {e}")
+
+    def file_equals(self, res: dict, file_node: dict) -> None:
+        logger.info(f"Response data : {res}")
+        logger.info(f"File Node with CCs : {file_node.props}")
+        errors = []
+        if res["hashes"]["md5"] != file_node.props["md5sum"]:
+            errors.append(
+                f"md5 value mismatch: '{res['hashes']['md5']}' != '{file_node.props['md5sum']}'"
+            )
+        if res["size"] != file_node.props["file_size"]:
+            errors.append(
+                f"file_size value mismatch: '{res['size']}' != '{file_node.props['file_size']}'"
+            )
+        if "urls" not in res.keys():
+            errors.append(f"urls keyword missing in {res.keys()}")
+        if "urls" in file_node.props.keys():
+            if file_node.props["urls"] not in res["urls"]:
```

Comment:
should there be a `for url in file_node.props["urls"]` statement here? like for `authz`

Reply:
file_node.props["urls"] is a string and res["urls"] is a list, hence not using a for loop as in authz

Reply:
then `file_node.props["urls"]` should be changed to a list to match what indexd supports, and to match the plural naming  "urlS", to avoid confusion